code/modules/mob/living/carbon/alien/diona/life.dm

```diff
@@ -34,6 +34,10 @@
 		check_status_as_organ()
 
 	if (!gestalt)
+		if(master_nymph && !client && master_nymph != src)
+			walk_to(src,master_nymph,1,movement_delay())
```

Comment:
Walk to is bad juju. It stops clean deletion. Make sure these nymphs call `walk_to(src, 0)` in their Destroy() proc. Or somewhere up the parent tree.